README.md

```diff
@@ -98,63 +108,56 @@ CSS
     background: rgba(255, 255, 255, 0.5);
 }
 ```
-- Click **Apply**.
```

Comment:
Why removig this?

Reply:
Moved to make it more clear (not removed).

Reply:
Didn't saw that either. woops.

Reply:
:)